libs/component-library/src/lib/bottom-sheet/bottomSheetModal.tsx

```diff
@@ -100,7 +90,12 @@ export const FxBottomSheetModal = React.forwardRef<
               {title}
             </FxText>
```

Comment:
It potentially makes sense to remove this title text of the bottom sheet since not all bottom sheets use this text style / sizing

Reply:
We could make it an optional prop